frontend/src/components/GovtMasthead/GovtMasthead.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { useDisclosure } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import {
+  GovtMasthead as GovtMastheadComponent,
+  GovtMastheadProps,
+} from './GovtMasthead'
+
+export default {
+  title: 'Components/GovtMasthead',
+  component: GovtMastheadComponent,
+  decorators: [],
+} as Meta
+
+const Template: Story<GovtMastheadProps> = (args) => {
+  const props = useDisclosure(args)
```

Comment:
not sure why we cannot click to open/close the masthead in the story, it's quite important behaviour to be able to test; we should fix that